web-admin/src/routes/[organization]/-/settings/+layout.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import { page } from "$app/stores";
+  import LeftNav from "@rilldata/web-admin/components/nav/LeftNav.svelte";
+
+  $: organization = $page.params.organization;
+  $: basePage = `/${organization}/-/settings`;
+
+  const navItems = [
+    {
+      label: "General",
+      route: "general",
+    },
+    {
+      label: "Billing",
+      route: "billing",
+    },
+    {
+      label: "Usage",
+      route: "usage",
+    },
+  ];
+</script>
+
+<LeftNav
+  title="Settings"
+  {basePage}
+  baseRoute="/[organization]/-/settings"
+  {navItems}
+>
+  <slot />
+</LeftNav>
```

Comment:
Rather than the `main` page content being a child of `LeftNav`, I think it'd be more conventional for the `main` page content and `LeftNav` component to be siblings, like:
```
<div class="layout-container">
  <LeftNav {title} {basePage} {baseRoute} {navItems} />
  <main>
    <slot />
  </main>
</div>
```

Reply:
With this approach, it'd also make sense to pull the "Settings" header out of the `LeftNav` component.